doc/source/database_structure.rst

```diff
@@ -74,22 +74,18 @@ In order to clear and rebuild the cache (i.e., reset it), you can simply run
 Query language
 ^^^^^^^^^^^^^^
 
-Since version `0.3` there is a query language in place for the searching
-of documents.
-The queries can contain any field of the info file, e.g.,
-``author:einstein publisher : review`` will match documents that have
-a matching ``author`` with ``einstein`` AND have a ``publisher``
-matching ``review``.
-The AND part here is important, since
-only the ``AND`` filter is implemented in this simple query
-language. At the moment it is not possible to do an ``OR``.
-If you need this, you should consider using the
-`Whoosh database`_.
+Since version ``v0.3`` there is a query language in place for the searching of
+documents. The queries can contain any field of the info file, e.g.,
+``author:einstein publisher : review`` will match documents that have ``author``
+match with ``einstein`` AND ``publisher`` match with ``review``. The AND part
+here is important, since only the ``AND`` filter is implemented in this simple
+query language. At the moment it is not possible to do an ``OR``. If you need
+this, you should consider using the `Whoosh database`_.
 
 For illustration, here are some examples:
 
   - Open documents where the author key matches 'albert' (ignoring case) and
-    year matches '19' (i.e., 1990, 2019, 1920):
+    year matches '05' (i.e. could be '1905' or '2005'):
```

Comment:
These examples were tying together with an Einstein publication in the year `1905`, and so there was a logical mistake here without them all using the number `05`